src/workspaces/reducers.ts

```diff
@@ -21,6 +24,7 @@ const initialState: WorkspacesState = {
   isDeleteWorkspaceFormOpen: false,
   isAddVocabularyFormOpen: false,
   workspaces: {},
+  tools: zipObject(tools.map( t => t.url ), tools)
```

Comment:
I can't see a reason of putting the tools into the store... perhaps to make them configurable in the future? I guess it's fine now.

Reply:
Yeah exactly. A bit overkill now, but was thinking about some configuration of the tools in the future.